gulpfile.js

```diff
@@ -297,6 +302,218 @@ gulp.task('minifyRelease', ['generateStubs'], function() {
     });
 });
 
+gulp.task('deploy', function(done) {
+    var cacheControl = "max-age=3600";
+    var concurrencyLimit = 2000;
+
+    var argv = yargs.usage('Usage: delpoy -b [Bucket Name] -d [Upload Directory]')
+        .demand(['b', 'd']).argv;
+
+    var uploadDir = argv.d;
+    var bucketName = argv.b;
+
```

Comment:
We should prompt for confirmation with the provided details before doing anything.  The readline module (which is included with node) can be used for this:

```javascript
    var iface = readline.createInterface({
        input: process.stdin,
        output: process.stdout
    });

    iface.question("Question", function (answer) {

    });
```